tools/restatectl/src/commands/log/reconfigure.rs

```diff
@@ -95,7 +96,34 @@ async fn reconfigure(connection: &ConnectionInfo, opts: &ReconfigureOpts) -> any
             segment_index: opts.segment_index,
             params,
         })
-        .await?;
+        .await?
+        .into_inner();
+
+    c_println!("✅ Log reconfiguration");
+    c_println!(" └ Segment Index: {}", response.new_segment_index);
+    let Some(sealed_segment) = response.sealed_segment else {
+        return Ok(());
+    };
+
+    c_println!();
+    c_println!("Sealed segment");
+    c_println!(" ├ Index: {}", response.new_segment_index - 1);
+    c_println!(" ├ Tail LSN: {}", sealed_segment.tail_offset);
+    c_println!(" ├ Provider: {}", sealed_segment.provider);
+    let Ok(provider) = ProviderKind::from_str(&sealed_segment.provider, true) else {
+        c_eprintln!("Unkown provider type '{}'", sealed_segment.provider);
+        return Ok(());
```

Comment:
Would it be appropriate to return `Err` on this path?

Reply:
I was thinking about this actually, and decided not too because the seal_and_extend call actually succeeded, but for some reason the previous provider kind is not known to the restatectl command. Maybe because of a version mismatch with the server.

But hence the operation actually succeeded, I still return `Ok(())`